.github/workflows/cpu_unittest.yml

```diff
@@ -0,0 +1,158 @@
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      matrix:
+        # os: [macos-latest, windows-latest, ubuntu-latest]
+        os: [ubuntu-latest]
+        python-version: [3.7, 3.8, 3.9]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Setup Miniconda
+        uses: conda-incubator/setup-miniconda@v2.0.0
+        with:
+          auto-update-conda: true
+          python-version: "${{ matrix.python-version }}"
+      - name: unit-test
+        shell: bash -l {0}
+        run: |
+          conda env create -n unittest -f .github/workflows_env/unittest_env.yml
+          conda activate unittest
+          conda list
+          python3 -m pip install --upgrade -e common/[tests]
+          python3 -m pip install --upgrade -e core/[all,tests]
+          env
+          cd core/
+          python3 -m pytest --junitxml=results.xml --runslow tests
+  features:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      matrix:
+        # os: [macos-latest, windows-latest, ubuntu-latest]
+        os: [ubuntu-latest]
+        python-version: [3.7, 3.8, 3.9]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Setup Miniconda
+        uses: conda-incubator/setup-miniconda@v2.0.0
+        with:
+          auto-update-conda: true
+          python-version: "${{ matrix.python-version }}"
+      - name: unit-test
+        shell: bash -l {0}
+        run: |
+          conda env create -n unittest -f .github/workflows_env/unittest_env.yml
+          conda activate unittest
+          conda list
+          python3 -m pip install --upgrade -e common/[tests]
+          python3 -m pip install --upgrade -e features/
+          env
+          cd features/
+          python3 -m pytest --junitxml=results.xml --runslow tests
+  tabular:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash
+    strategy:
+      matrix:
+        # os: [macos-latest, windows-latest, ubuntu-latest]
+        os: [ubuntu-latest]
+        python-version: [3.7, 3.8, 3.9]
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Setup Miniconda
+        uses: conda-incubator/setup-miniconda@v2.0.0
+        with:
+          auto-update-conda: true
+          python-version: "${{ matrix.python-version }}"
+      - name: unit-test
+        shell: bash -l {0}
+        run: |
+          conda env create -n unittest -f .github/workflows_env/unittest_env.yml
+          conda activate unittest
+          conda list
+          python3 -m pip install --upgrade -e common/[tests]
+          python3 -m pip install --upgrade -e core/[all]
+          python3 -m pip install --upgrade -e features/
+          python3 -m pip install --upgrade -e tabular/[all]
+          python3 -m pip install --upgrade -e text/
+          python3 -m pip install --upgrade -e vision/
```

Comment:
not sure if we need this for `tabular`

Reply:
We are offloading cpu tests to github action, that's why I put tabular here

Reply:
Alex is referring to the installation of text and vision modules

Reply:
Oh, the tabular CI will fail while installing autogluon without text and vision

Reply:
Why? Isn't this CPU only?

Reply:
NVM, that's actually a mistake. Updated it